R/overline.R

```diff
@@ -314,7 +314,19 @@ overline2 <-
       if (nrow(sl) > regionalise) {
         message(paste0("large data detected, using regionalisation, nrow = ", nrow(sl)))
         suppressWarnings(cents <- sf::st_centroid(sl))
-        grid <- sf::st_make_grid(cents, what = "polygons")
+        # Fix for https://github.com/r-spatial/sf/issues/1777
+        if(sf::st_is_longlat(cents)){
+          bbox <- sf::st_bbox(cents)
+          bbox[1] <- bbox[1] - abs(bbox[1] * 0.001)
```

Comment:
I like the cautious approach, 0.1% increase is a minimal change, cannot foresee any side effects from this.